types/jquery/JQuery.d.ts

```diff
@@ -8703,7 +8703,7 @@ $( "span" ).click(function() {
      */
     parents<K extends keyof HTMLElementTagNameMap>(selector: K | JQuery<K>): JQuery<HTMLElementTagNameMap[K]>;
     parents<K extends keyof SVGElementTagNameMap>(selector: K | JQuery<K>): JQuery<SVGElementTagNameMap[K]>;
-    parents(selector?: JQuery.Selector): this;
+    parents<E extends HTMLElement>(selector?: JQuery.Selector): JQuery<E>; // tslint:disable-line
```

Comment:
We don't allow global lint removal for a line in DT, but you can disable the exact error you see here 👍🏻

Reply:
Thanks for the feedback. For some reason I was under the impression `npm run lint` told me which three disable-rules are allowed. Guess I was wrong. Would it be possible to add a lint rule for not allowing global lint removal?